### DISCUSSION TITLE
Remove `isFloat`

### DISCUSSION THREAD

**Discussion by aleclarson** [radashi-org/radashi#111] [Ideas]
I'm planning to remove `isFloat` (which checks if the number is not an integer) as this is an uncommon need.

**Comment by AQian0** [comment]
I think it's ok.
JavaScript doesn't have int and float like other languages, it really only has number. From that perspective, distinguishing whether it's a float or not is less meaningful for the program itself, and more on the business side. In that case, they should be left to the developer to implement, not the tool library.

（All of the above is from translation software.）
